### PR TITLE
[MWPW-154733] Added early returns and error handing for invalid URLs

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Adds checks to utils.js for empty URLs. Some functions tried to construct new URLs with empty hrefs.

Testing Instructions

Resolves: https://jira.corp.adobe.com/browse/MWPW-154733

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://url-sanitization--express--adobecom.hlx.page/br/express/feature/video/resize

### CODE REVIEW

**Review by vhargrave — APPROVED** [review]
very nice!
tested here too since we were just having errors here - https://url-sanitization--express--adobecom.hlx.page/jp/express/learn/blog/line-stickers-campaign

File: express/scripts/utils.js [hunk 1]
```diff
@@ -473,7 +472,9 @@ export function getIconElement(icons, size, alt, additionalClassName, altSrc) {
 }
 
 export function transformLinkToAnimation($a, $videoLooping = true) {
-  if (!$a || !$a.href.endsWith('.mp4')) {
+  if (!$a || !$a.href || !$a.href.endsWith('.mp4')) {
+    console.error('Invalid video URL');
```
Comment (JingleH) [inline express/scripts/utils.js:476]: can we clean up the console messages for this line and the lines below?

**Review by JingleH — CHANGES_REQUESTED** [review]

File: express/scripts/utils.js (hunk 1 above)
Comment (echen-adobe) [reply express/scripts/utils.js:476]: I think there should be some indication that an incorrect link was authored.

Comment (JingleH) [reply express/scripts/utils.js:476]: i agree on this point too, but why would you want to log it into end users' devices? If you want to debug in production, you should log it to logging service like LANA. If you want to notify authors during authoring, you can detect the dev/stage environment and either crash the whole block and display an error message in the page or send an alert. Logging things in the browser in most cases is only useful for development and post that, it seems less professional.

**Review by JingleH — APPROVED** [review]

**Pull request merged**